prebuilt-tasks/pom.xml

```diff
@@ -77,5 +77,17 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>dev.parodos</groupId>
+            <artifactId>notification-service-sdk</artifactId>
+            <version>${revision}</version>
+            <scope>compile</scope>
```

Comment:
`compile` is the default scope and can be spared.

Reply:
Removed